client/src/main/java/io/pravega/client/control/impl/ControllerImpl.java

```diff
@@ -1571,8 +1571,8 @@ private void closeChannel() {
         final long requestId = requestIdGenerator.get();
         long traceId = LoggerHelpers.traceEnter(log, "updateReaderGroup", rgConfig, requestId);
 
-        final CompletableFuture<UpdateReaderGroupStatus> result = this.retryConfig.runAsync(() -> {
-            RPCAsyncCallback<UpdateReaderGroupStatus> callback = new RPCAsyncCallback<>(requestId, "updateReaderGroup", scope, rgName, rgConfig);
+        final CompletableFuture<UpdateReaderGroupResponse> result = this.retryConfig.runAsync(() -> {
```

Comment:
why are we changing the name of this class `UpdateReaderGroupStatus`?

Reply:
UpdateReaderGroupResponse has 'Status' of the operation and 'generation' post upgrade. Hence the change in name.